### DISCUSSION TITLE
Performance monitoring

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#157] [Ideas]
Hi,
I suggest for better tracking either bridge or camera, or wifi coverage,  to add performance GUI (or any easily understanding format)...am not expert with ready made codes, but I can see one PProf...anyways, the aim to track and log main key perfoamnce indictators...like, uptime, bitrate, FPS, wifi signal, ...etc.

**Comment by mrlt8** [comment]
This might be possible via MQTT?

**Comment by AdiAbuAli** [comment]
Am not familiar MQTT yet....let me try to play around and update you

Thanks,
________________________________
